### PR TITLE
drivers/{b,c}*: model modules in Kconfig

### PR DESCRIPTION
**Pull request by leandrolanzieri**
### Contribution description
This models some more high-level driver modules in Kconfig. A change in Makefile.include was needed to allow to activate packages via Kconfig when `TEST_KCONFIG=1`.

### Testing procedure
- Check that the dependencies are correctly modelled, comparing to the current Makefiles.
- Check that the test applications of the correspondent drivers are added to the list of the CI
- Green CI

### Issues/PRs references
None

### CODE REVIEW

File: tests/driver_bmx055/app.config.test [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# this file enables modules defined in Kconfig. Do not use this file for
+# application configuration. This is only needed during migration.
+# include and auto-initialize all available sensors
+CONFIG_MODULE_PHYDAT=y
+CONFIG_MODULE_SAUL=y
+CONFIG_MODULE_SAUL_REG=y
+CONFIG_MODULE_SAUL_DEFAULT=y
```
Comment (fjmolinas) [inline tests/driver_bmx055/app.config.test:7]: When I use `menuconfig` I can see that `CONFIG_MODULE_SAUL_GPIO` is selected but I cant find where it is selected in the dependencies.

**Review by fjmolinas — COMMENTED** [review]
Just one question otherwise everything looks good.

File: tests/driver_bmx055/app.config.test (hunk 1 above)
Comment (fjmolinas) [reply tests/driver_bmx055/app.config.test:7]: Ahh dumb me: `default y if HAVE_SAUL_GPIO`

**Review by fjmolinas — APPROVED** [review]
ACK.

**Comment by leandrolanzieri** [comment]
Thanks for reviewing !

**Pull request merged**